packages/sdk-js/src/GrowthBook.ts

```diff
@@ -488,12 +498,48 @@ export class GrowthBook<
 
     // Apply new changes
     if (result.inExperiment) {
-      const undo = this._applyDOMChanges(result.value);
-      if (undo) {
-        this._activeAutoExperiments.set(experiment, {
-          undo,
-          valueHash,
-        });
+      if (result.value.urlRedirect && experiment.urlPatterns) {
+        const url = experiment.persistQueryString
+          ? mergeQueryStrings(this._getContextUrl(), result.value.urlRedirect)
```

Comment:
Worth calling out in the documentation that if SDK is not in a browser environment users will need to use setUrl() to set a context url. Only really necessary if users are looking to user persistQueryString on redirect experiments